examples/creatives/simid_overlay.js

```diff
@@ -39,4 +40,16 @@ class SimidOverlay extends BaseSimidCreative {
     document.getElementById(elementName).addEventListener(
         'click', sendMessageFunction.bind(this));
   }
+
+  sendMessageOnLog_(buttonElement, message) {
```

Comment:
I see you're kinda taking from the style of the function above, but I think you might be overcomplicating it for this use case a bit.
This function is only being used for the log button, so I don't think there is any need to take in the buttonElement as a param (it will only ever be log_button anyway).

Reply:
yes- I kept it here for uniform, but I can get rid of the buttonElement parameter